From deb013d93c3ce78891386637d6b3300289130df6 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Sun, 7 Aug 2016 22:28:50 +0100
Subject: Make build reproducible

Remove build date from the binary so that two builds on different dates
are binary identical.

Origin: upstream, https://github.com/hugosantos/mrd6/commit/14f27d5528a3e4c8e5369bdee0e35961a2491661
Forwarded: https://github.com/hugosantos/mrd6/pull/30
Last-Update: 2016-08-09
Applied-Upstream: commit: 14f27d5528a3e4c8e5369bdee0e35961a2491661
---
 src/Makefile | 1 -
 src/mrd.cpp  | 2 --
 2 files changed, 3 deletions(-)

--- a/src/Makefile
+++ b/src/Makefile
@@ -185,7 +185,6 @@ endif
 $(MRD_VERSION_CPP): $(SOURCES) Makefile Makefile.options
 	@set -e; mkdir -p $(dir $@); \
 		echo '/* This file is automatically generated */' > $(MRD_VERSION_CPP); \
-		echo 'const char *BuildDate = "$(NOW)";' >> $(MRD_VERSION_CPP)
 
 $(MODULES_CPP): Makefile Makefile.options
 	@set -e; mkdir -p $(dir $@); \
--- a/src/mrd.cpp
+++ b/src/mrd.cpp
@@ -75,7 +75,6 @@
 
 mrd *g_mrd = 0;
 
-extern const char *BuildDate;
 static const char *VersionInfo = "mrd6 0.10.0";
 
 static const char *defaultconffiles[] = {
@@ -2335,7 +2334,6 @@ bool mrd::socket_regs(base_stream &out,
 
 void mrd::show_base_info(base_stream &out) const {
 	out.xprintf("Version: %s\n", VersionInfo);
-	out.xprintf("Build date: %s\n", BuildDate);
 }
 
 bool mrd::show_info(base_stream &out, const std::vector<std::string> &ctx) {
